src/index.html

```diff
@@ -5,9 +5,60 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin ="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+          class="card__img"
+      >
+
+      <h1 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <p class="card__description">
+        Product code: 195434
+      </p>
+
+      <div class="card__stars-container">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <p class="reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="price-name">
+          Price:
+        </p>
+
+        <p class="price-amount">
```

Comment:
```suggestion
      <div class="card__price">
        <p class="card__price-name">
          Price:
        </p>

        <p class="card__price-amount">
```
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/ad2312e3-ccb4-4fe5-94b7-646e8eadc083)